src/style.css

```diff
@@ -1 +1,25 @@
-/* add styles here */
+body {
+  margin: 0;
+}
```

Comment:
According to the checklist, avoid using tag names for styling. It's recommended to style elements using classes. However, the `body` tag is an exception to this rule, so this is just a general reminder for the rest of your CSS.